focus-ios/Blockzilla/Search/SearchEngineManager.swift

```diff
@@ -51,7 +51,7 @@ class SearchEngineManager {
     }
 
     func hasDisabledDefaultEngine() -> Bool {
-        return getDisabledDefaultEngineNames().count > 0
+        return getDisabledDefaultEngineNames().isEmpty
```

Comment:
`getDisabledDefaultEngineNames().count > 0`, meaning that we want to know if we  have values, so the equivalent is `!getDisabledDefaultEngineNames().isEmpty`.